examples/Default.example

```diff
@@ -3,41 +3,66 @@ class Example extends React.Component {
   constructor(props) {
     super(props);
     cookie('accepts-cookies', '');
-    this.state = {
```

Comment:
It's broken only in the showroom as it listens to the scrolling of `window` which is never fired because Showroom has position `absolute`.

I'm trying to fix this in a separate branch, If I succeed we may be able to re-enable this.